File: Popsicles/Popsicle.c
#include<stdio.h>
int main(void){
    int siblings, popsicles;
    printf("Enter number of siblings: \n");
    scanf("%d", &siblings);
    printf("Enter number of Popsicles: \n");
    scanf("%d", &popsicles);
    if(popsicles % siblings != 0)
    printf("eat them yourself\n");
    else
    printf("give away\n");
}

File: Easy-exercises/Popsicles/Popsicle.php
<?php
// done by Leonard Appelgryn(KAPTAINCS3)
    echo ("Enter number of siblings: \n");
    $siblings = (int)readline("");
    echo ("Enter number of Popsicles: \n");
    $popsicles = (int)readline("");
    if($popsicles % $siblings != 0)
    echo ("eat them yourself\n");
    else
    echo ("give away\n");
?>